docs/dashboards/data-mathmodifier.md

```diff
@@ -0,0 +1,210 @@
+Data's MathModifier
```

Comment:
This Data?
![image](https://github.com/highcharts/highcharts/assets/7478772/41e41825-0344-4e2c-997e-f97b38f4fa81)

Maybe just "MathModifer" or "Data module: MathModifier" would be clearer.